services/dask-sidecar/src/simcore_service_dask_sidecar/worker.py

```diff
@@ -76,14 +79,21 @@ async def dask_setup(worker: distributed.Worker) -> None:
                     RabbitMQPlugin(settings.DASK_SIDECAR_RABBITMQ), catch_errors=False
                 )
             except Exception:
-                await worker.close()
+                await worker.close(reason="failed to add RabbitMQ plugin")
                 raise
+        try:
```

Comment:
MINOR: since you use it several times, perhaps you can create some helper to savely add plugins?

```python
async def add_plugin(worker, plugin):
    try:
        await worker.plugin_add(plugin, catch_errors=False)
    except Exception:
        await worker.close(reason=f"failed to add {plugin.name} plugin")
        raise

```